woocommerce/woocommerce
# Update get_recommended_themes() to check filtered response for active themes

Updates get_recommended_themes() function so that it checks all of the filtered themes to see if any are currently active so we can display the badge on the front end.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #40916.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Update WooCommerce with this branch, and make sure the `customize-store` feature flag is enabled.
2. Download [this gist](https://gist.github.com/crunnells/4940b3edf5377f796ffa0ed1d948fbd1) and install it as a plugin in wp-admin
3. Activate the uploaded `` plugin
4. Set the theme to Twenty Twenty Three
5. Go to `/wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store` and verify that Twenty Twenty Three shows the `Active theme` badge.
